### PR TITLE
add OnePile.app :latest

### PR DESCRIPTION
**Pull request by andre68723**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/onepile.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'onepile' do
+  version ':latest'
+  sha256 :no_check
+
+  url 'https://onepile.app/download-macos'
```
Comment (vitorgalvao) [inline Casks/onepile.rb]: `https://onepile.app/macos/OnePile-#{version}.zip`

File: Casks/onepile.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'onepile' do
+  version ':latest'
```
Comment (vitorgalvao) [inline Casks/onepile.rb]: `2.0-1092`

File: Casks/onepile.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/onepile.rb]: `sha256` also needs to be set. You can find what it is by running `shasum --algorithm 256` on the downloaded file.

File: Casks/onepile.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/onepile.rb]: `appcast 'https://onepile.app/sparklecast.xml'`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
> * [x]  Attempted to find an [appcast](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Did you really try this? Because it was really easy to find. If you didn’t, I’d rather you didn’t tick the box.

**Review by vitorgalvao — APPROVED** [review]

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
You just indeed your changes. Don’t force push, it makes it harder to review.

File: Casks/onepile.rb [hunk 4]
```diff
@@ -1,8 +1,9 @@
 cask 'onepile' do
-  version ':latest'
-  sha256 :no_check
+  version '2.0-1092'
+  sha256 '8deb9355b80cc8ddc14797db98bdb917bae07fb2c92d785e797e19f6a8a30b32'
 
-  url 'https://onepile.app/download-macos'
+  url 'https://onepile.app/update/macos/OnePile-2.0-1092.zip'
```
Comment (vitorgalvao) [inline Casks/onepile.rb]: Missing the interpolation.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/onepile.rb (hunk 1 above)
Comment (andre68723) [reply Casks/onepile.rb]: thanks, did not locate it. now added

File: Casks/onepile.rb (hunk 4 above)
Comment (andre68723) [reply Casks/onepile.rb]: do you mean 
`url 'https://onepile.app/update/macos/OnePile-2.0-1092.zip'` or 
`url 'https://onepile.app/update/macos/OnePile-{version}.zip' ` ?

**Pull request merged**